theme/static/js/profile.js

```diff
@@ -2,12 +2,46 @@
 * Created by Mauriel on 3/9/2017.
 */
 
+var CHECK_PROFILE_IMAGE_SIZE = true;
+var PROFILE_IMAGE_SIZE_BYTE = 64000;
+var KILO_BYTE="KB";
+var PROFILE_IMAGE_SIZE_K = PROFILE_IMAGE_SIZE_BYTE/1000 + " " + KILO_BYTE;
+
+// avoid tying error, define it as variable
+var errorElementID = "errorSizeLoadFile";
+
+function clearSizeErrorMsg() {
+    $("#" + errorElementID).remove();
+}
+
+function displaySizeErrorMsg() {
+     let errorMsg = "Error: Photo size can not exceed " + PROFILE_IMAGE_SIZE_K;
```

Comment:
We need to also clear the value of the file input here. Otherwise users can just click on Save Changes and the picture will go through anyways.

Reply:
Also, the back end will need to check for the image size too and validate it. @pkdash - any thoughts?

Reply:
> We need to also clear the value of the file input here. Otherwise users can just click on Save Changes and the picture will go through anyways.

What you said proved to be true. The oversized image did get saved even thought I abort the reader. How can this be... Is the file still cached somewhere?

Reply:
My guess would be that the reader is running after the file is cached in the file input.

Reply:
Got it fixed. 
`input.value="" ` does the trick